Backend/src/database/models/institution.ts

```diff
@@ -0,0 +1,57 @@
+import {Document, model, Schema} from "mongoose";
+
+export interface IInstitution extends Document {
+   name:string,
+   country:string,
+   profilePictureURL: string,
+   backgroundPictureURL: string,
+   adminIds: [Schema.Types.ObjectId],
+   invitedMemberIds: [Schema.Types.ObjectId],
+   projects: [Schema.Types.ObjectId],
```

Comment:
Shouldn't we here also include memberIds not just ones who are invited.

Reply:
yes, member ids field will be added.